bot/client.js

```diff
@@ -6,10 +6,17 @@ const { isAdmin } = require('./botUtils.js');
 if (process.env.NODE_ENV === 'production' && !process.env.DEBUG_MODE) process.env.DEBUG_MODE = 0;
 
 const client = new Discord.Client();
-
+const channelId = '442034467482370052';
```

Comment:
Unable to test functionality because Channel ID is only specific to your server, this needs to be made dynamic